code/game/objects/structures/crates_lockers/crates/gear_loadout.dm

```diff
@@ -661,3 +661,57 @@
 	new /obj/item/gun/projectile/pistol(src)
 	new /obj/item/ammo_magazine/mc9mm(src)
 	new /obj/item/ammo_magazine/mc9mm(src)
+
+//Vault Gear for hardsuit token
+/obj/structure/closet/crate/secure/gear_loadout/vault
+	var/associated_vault_hardsuit = /obj/item/rig/hazard
+/obj/structure/closet/crate/secure/gear_loadout/vault/hazard
+	associated_vault_hardsuit = /obj/item/rig/hazard
```

Comment:
Indentation is funky throughout this file, please fix that. Generally add a newline if it's missing after a definition.

Reply:
As we discussed over discord, I'll remove the paragon module, and the sol one. I don't understand the issue with the indentation. Could you elaborate or show me an example of how it should look? I just copy and pasted the layout from the normal hardsuit token.